apps/etterlatte-behandling/src/main/kotlin/behandling/etteroppgjoer/EtteroppgjoerRoutes.kt

```diff
@@ -117,8 +117,8 @@ fun Route.etteroppgjoerRoutes(
                                 etteroppgjoerId,
                                 brukerTokenInfo,
                             )
+                            forbehandlingService.ferdigstillForbehandling(etteroppgjoerId, brukerTokenInfo)
```

Comment:
Jeg tenker vi bør ha denne før brev-kallet, hvis ikke så vil vi sende avgårde brevet før vi bailer ut med f.eks. vi har ikke brukt nyeste pgi

Reply:
Burde vi før ferdigstilling av brev sjekke at forbehandling er ferdigstilt?

Reply:
Jeg vet ikke om det blir helt nødvendig -- men det er nok lurt å ta en ekstra sjekk over flytene våre i ferdigstilling f.eks. og se om det er feilsituasjoner vi bør fange opp. Så jeg ser for meg at vi kanskje legger det til senere som en helhetlig vurdering? 

Noen sjekker kan plutselig komme litt i veien hvis noe annet har gått galt også 😅